framework/Util/TDbParameterModule.php

```diff
@@ -233,14 +233,19 @@ protected function createDbTable()
 		$db = $this->getDbConnection();
 		$driver = $db->getDriverName();
 		$autoidAttributes = '';
+		$autotype = 'INTEGER';
 		$postIndices = '; CREATE UNIQUE INDEX tkey ON ' . $this->_tableName . '(' . $this->_keyField . ');' .
 		($this->_autoLoadField ? ' CREATE INDEX tauto ON ' . $this->_tableName . '(' . $this->_autoLoadField . ');' : '');
 		if ($driver === 'mysql') {
```

Comment:
use switch() instead?
The default branch could either default to mysql or throw an exception.

Reply:
good point

Reply:
corrected